### PR TITLE
DBZ-2005 Implement SKIPPED_OPERATIONS for postgres

### PR DESCRIPTION
**Pull request by blcksrx**

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java [hunk 1]
```diff
@@ -163,6 +165,15 @@ else if (message.getOperation() == Operation.COMMIT) {
                         offsetContext.updateWalPosition(lsn, lastCompletelyProcessedLsn, message.getCommitTime(), message.getTransactionId(), tableId,
                                 taskContext.getSlotXmin(connection));
 
+                        Set<Envelope.Operation> skippedOperations = taskContext.getConfig().getSkippedOps();
+                        if (skippedOperations != null) {
+                            for (Envelope.Operation operation : skippedOperations) {
+                                if (operation == Envelope.Operation.CREATE || operation == Envelope.Operation.UPDATE || operation == Envelope.Operation.DELETE) {
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java:171]: Hum, this doesn't really make sense; we'd skip *any* record as soon as there's a single skipped op configured. Also, this should go into `EventDispatcher`, which allows to implement this feature in a single place (MySQL was a bit special, as it isn't using the framework structure around the dispatcher yet).

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java [hunk 2]
```diff
@@ -527,6 +527,25 @@ public void shouldReceiveChangesForInsertsIndependentOfReplicaIdentity() throws
...
+                .with(PostgresConnectorConfig.SKIPPED_OPERATIONS, "d")
+                .with(PostgresConnectorConfig.SNAPSHOT_MODE, SnapshotMode.NEVER),
+                false);
+        consumer = testConsumer(0);
+
+        // Insert
+        TestHelper.execute("INSERT INTO skipped_op_table (data) VALUES ('test')");
+        consumer.expects(1);
+
+        // Delete
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java:543]: It's important to assert on a nother non-skipped record after that. I'd suggest to simply follow the same test flow I used for MySQL.

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java (hunk 1 above)
Comment (blcksrx) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java:171]: I couldn't find `EventDispatcher` where is it?

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java (hunk 2 above)
Comment (blcksrx) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java:543]: Thanks. I'm going to do that

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java:171]: In the core module, see `io.debezium.pipeline.EventDispatcher`.

**Pull request closed without merging**